### ISSUE TITLE
BufferGeometryUtils.mergeBufferGeometries() does not always yield error when attributes aren't consistent

### ISSUE THREAD

**Issue by marquizzo** [mrdoob/three.js#19193]
`BufferGeometryUtils.mergeBufferGeometries()` requires all geometries to have compatible attributes to be successful. This [yields an error as expected](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/utils/BufferGeometryUtils.js#L224) if the first geometry has less attributes than subsequent geometries, but it does not yield an error if the first geometry has **more** attributes than subsequent geometries. Here's a simple example:

```
const geom1 = new THREE.TorusBufferGeometry( 4, 0.5, 16, 100 );
const geom2 = new THREE.BoxBufferGeometry(5, 5, 5);
const geom3 = new THREE.SphereBufferGeometry(3, 32, 32);

// Deleting attribute on second geom does not yield error
geom2.deleteAttribute("normal");

const mergedGeom = BufferGeometryUtils.mergeBufferGeometries([geom1, geom2, geom3], false);
const mergedMesh = new THREE.Mesh(
  mergedGeom,
  new THREE.MeshNormalMaterial()
);

this.scene.add(mergedMesh);
```

You can see in this live demo that deleting the normals for the `Box` ruins the normals for the `Sphere` because it continues without an error: https://codepen.io/marquizzo/pen/eYpgBdL

![normals](https://user-images.githubusercontent.com/7864858/80043879-c065cc80-84b7-11ea-9b66-8cb233a55c5e.jpg)

This is because we're only testing [if the current geometry's attributes exist in the first geometry](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/utils/BufferGeometryUtils.js#L222), but not the other way around.

##### Three.js version

- [ x ] Dev

##### Browser

- [x] All of them

##### OS

- [x] All of them

**Comment by WestLangley** [comment]
three.js does not, in general, validate inputs. An exception is when an API is changed, and validation seems preferable.

If you can fix it easily, then fine -- especially since there is already some attempt to flag errors.

However, given the three.js convention, it is also reasonable to simply state the requirements explicitly in the docs -- and also via inline comments in the code.

**Comment by marquizzo** [comment]
@WestLangley What do you think of this solution? https://github.com/mrdoob/three.js/pull/19208 It's a simple counter that compares to the first geometry's number of attributes.

**Comment by Mugen87** [comment]
Should be fixed via #19193 🎉
